### ISSUE TITLE
Trouble converting mesh from vtk to obj

### ISSUE THREAD

**Issue by Perpetual3134** [w1th0utnam3/splashsurf#175]
Hi there,

created a .vtk sequence with reconstruct and would like to convert it to obj...

.\splashsurf.exe convert --domain-max="1000;1000;1000" --domain-min="0;0;0" --mesh out\ParticleData_Fluid_surface_151.vtk -o obj_con\Surface_151.obj

Reading mesh from "out\ParticleData_Fluid_surface_151.vtk"...
[2023-04-17T12:12:58.423520+02:00][splashsurf][ERROR] Error occurred: Failed to load surface mesh from file "out\ParticleData_Fluid_surface_151.vtk"
[2023-04-17T12:12:58.423824+02:00][splashsurf][ERROR]   caused by: Expected only triangle cells. Invalid number of vertex indices (3) of cell 0

Could you please help?

Kind regards

**Comment by Perpetual3134** [comment]
Have seen issue #174... If you could implement obj to sequences that would work for me fine too

**Comment by w1th0utnam3** [comment]
Sorry, this was a bug that is fixed in the repository now but I did not make a new release since then.
Until I make a new release (hopefully soon) you can install the current version from this repository using
```
cargo install --git https://github.com/w1th0utnam3/splashsurf.git
```

**Comment by w1th0utnam3** [comment]
@Perpetual3134 I just published a new release (v0.9.0) which should fix the error and which also allows you to directly output .obj files from a sequence. If you use `--input-sequence=file_{}.vtk`, just specify additionally `--output-file=file_surace_{}.obj` and it should work. 

You can install the latest version of splashsurf normally using `cargo install splashsurf`.
